xm-commons-permission/src/main/java/com/icthh/xm/commons/permission/domain/mapper/PermissionMapper.java

```diff
@@ -47,7 +47,9 @@ public String permissionsToYml(Collection<Permission> permissions) {
      *
      * @param yml string
      * @return permissions map
+     * @deprecated use {@link com.icthh.xm.commons.permission.service.PermissionMappingService#ymlToPermissions}
      */
+    @Deprecated(forRemoval = true)
     public Map<String, Permission> ymlToPermissions(String yml) {
```

Comment:
pls provide update uaa ms
Uaa use this methods

Reply:
https://github.com/xm-online/xm-uaa/pull/163/files